plugins/task-plugin/package.json

```diff
@@ -60,7 +60,7 @@
     "backend": "lib/task-plugin-backend.js"
   },
   "dependencies": {
-    "inversify": "4.13.0",
+    "inversify": "4.14.0",
```

Comment:
We have approved CQ for 4.13.0 https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19424
Should we open CQ for 4.14.0?

Reply:
I think yes, but we need TypeA CQ check, I change that version to be aligned with Theia

Reply:
CQ - https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20319